assets/stylesheets/landing.css

```diff
@@ -63,24 +84,24 @@ body {
 
 .corner-nav-item.top-left {
```

Comment:
here also, use fixed pixel values, or `rem` value which is relative to root font size, and if you are going to make root font size dynamic using `calc()` with experimental `vw` values plus some px value, you will have fluid and responsive spacing.